packages/jaeger-ui/src/components/TracePage/ScrollManager.tsx

```diff
@@ -87,7 +87,7 @@ function isSpanHidden(span: Span, childrenAreHidden: Set<string>, spansMap: Map<
  */
 export default class ScrollManager {
   _trace: Trace | TNil;
-  _scroller: IScroller;
+  _scroller: IScroller | undefined;
```

Comment:
not a good change - messing with semantics of the type just to appease a linter. The constructor always takes a scroller, so this isn't an optional field

Reply:
It's this `destroy` function that is setting the same to be `undefined`.
- If this is intended behavior, we should allow it to be `undefined` as we are explicitly setting it to be.
- Otherwise we should remove this line from this function.

https://github.com/jaegertracing/jaeger-ui/blob/ec3aba07a6439800b7477e761a3feabd4e74d47c/packages/jaeger-ui/src/components/TracePage/ScrollManager.tsx#L269-L273

Reply:
don't really understand the point of this destroy() but ok, let's just keep it consistent with other fields and use TNil?

Reply:
Sounds good!